subprojects/maven/src/main/java/org/gradle/api/publish/maven/tasks/PublishToMavenRepository.java

```diff
@@ -110,53 +106,13 @@ private PublishSpec computeSpec() {
         if (repository == null) {
             throw new InvalidUserDataException("The 'repository' property is required");
         }
-
-        checkCredentialSafety(repository);
-
         MavenNormalizedPublication normalizedPublication = publicationInternal.asNormalisedPublication();
         return new PublishSpec(
                 RepositorySpec.of(repository),
                 normalizedPublication
         );
     }
 
-    private void checkCredentialSafety(DefaultMavenArtifactRepository repository) {
```

Comment:
Deleted much of the dead code that I left behind from the "unsafe credentials" kerfuffle, in order to minimize the delta to PublishToIvyRepository.